lib/fs.js

```diff
@@ -2479,3 +2479,454 @@ Object.defineProperty(fs, 'SyncWriteStream', {
   set: internalUtil.deprecate((val) => { SyncWriteStream = val; },
                               'fs.SyncWriteStream is deprecated.', 'DEP0061')
 });
+
+
+// The Promises API
+
+async function* readEverything(fd) {
+  let loops = 0;
+  let eof = false;
+  let position = 0;
+  const buffer = Buffer.allocUnsafe(kReadFileBufferLength);
+  while (!eof) {
+    if (loops > 5) process.exit(0);
```

Comment:
there is no place where `loop` is incremented

Reply:
oh, that's a left over... should have removed that.